tutorials/detect.ipynb

```diff
@@ -181,9 +244,10 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Measurements\n",
+    "## Li & Ma significance maps\n",
     "\n",
-    "* TODO: show cutout for a few sources and some aperture photometry measurements (e.g. energy distribution, significance, flux)"
+    "Additionaly we can compute significance for an observed number of counts and known background using equation (17) from the [Li & Ma (1983)](https://ui.adsabs.harvard.edu/abs/1983ApJ...272..317L/abstract). We can perform this calculation intergating the counts within different radius. To do so we use an astropy Tophat kernel with the `compute_lima_image` function.\n",
```

Comment:
Here you should say that this is an extension of the Li & Ma formula for known background.